assets/js/components/wp-dashboard/wp-dashboard-session-duration.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * WPDashboardSessionDuration component.
+ *
+ * Site Kit by Google, Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { prepareSecondsForDisplay } from '../../util';
+import DataBlock from '../data-block';
+import { STORE_NAME } from '../../modules/analytics/datastore/constants';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { overviewReportDataDefaults } from '../../modules/analytics/util';
+import PreviewBlock from '../PreviewBlock';
+import ReportError from '../ReportError';
+import calculateOverviewData from '../../modules/analytics/util/calculateOverviewData';
+const { useSelect } = Data;
+
+const WPDashboardSessionDuration = () => {
+	const {
+		data,
+		loading,
+		error,
+	} = useSelect( ( select ) => {
+		const store = select( STORE_NAME );
+
+		const commonArgs = {
+			dateRange: select( CORE_USER ).getDateRange(),
+		};
+
+		const args = {
+			...overviewReportDataDefaults,
+			...commonArgs,
+		};
+
+		return {
+			data: store.getReport( args ),
+			loading: ! store.hasFinishedResolution( 'getReport', [ args ] ),
+			error: store.getErrorForSelector( 'getReport', [ args ] ),
+		};
+	} );
+
+	if ( loading ) {
+		return <PreviewBlock width="48%" height="92px" />;
+	}
+
+	if ( error ) {
+		return <ReportError moduleSlug="analytics" error={ error } />;
+	}
+
+	const {
+		averageSessionDuration,
+		averageSessionDurationChange,
+	} = calculateOverviewData( data );
+
+	return (
+		<DataBlock
+			className="googlesitekit-wp-dashboard-stats__data-table overview-average-session-duration"
+			title={ __( 'Avg. Time on Page', 'google-site-kit' ) }
```

Comment:
I see this was this way in the IB, and is already written this way in the existing app (https://github.com/google/site-kit-wp/blob/develop/assets/js/modules/analytics/components/wp-dashboard/WPAnalyticsDashboardWidgetOverview.js#L115), but it's a shame because it's not very accessible (no `<def>` tag) and it isn't especially clear to translators what the abbreviation is.

Sure, in this case it's pretty easy to infer from context, but it might be worth improving this either now (if it isn't a UI issue to expand "Avg." to "Average" or later in a follow-up issue if it is...

Reply:
If I change it to `Average`, at least I don't see any issue.

![image](https://user-images.githubusercontent.com/72751210/101400828-796a0180-38f7-11eb-9027-226fa3a6ba3e.png)
vs
![image](https://user-images.githubusercontent.com/72751210/101400876-8686f080-38f7-11eb-9336-9007f3b3a456.png)

Let me know if I should change it 😃

Reply:
My immediate thought was yes, but I did consider consistency here. I had a look at what the actual Analytics page does and it's also "Avg." where "Average" would totally work:

<img width="616" alt="Screenshot 2020-12-08 at 16 18 20" src="https://user-images.githubusercontent.com/90871/101509829-174de280-3971-11eb-9fee-5fd9900d738a.png">

Let's stick with "Avg." for now and see about proposing an upstream fix then mirroring the site's change. I think the consistency in labelling is useful so we'll stick with the (admittedly less-good) one for now. 😅